js/ui/interaction.js

```diff
@@ -201,6 +201,8 @@ function Interaction(el) {
             if (value !== 0 && (value % 4.000244140625) === 0) {
                 // This one is definitely a mouse wheel event.
                 type = 'wheel';
+                // Normalize this value to match trackpad.
+                value = +(value/4).toFixed();
```

Comment:
what's the purpose of calling `toFixed` and then converting back to number here?